pkg/build/types/parser/deffile.go

```diff
@@ -417,12 +417,15 @@ func doHeader(h string, d *types.Definition) error {
 // and parse it into a Definition struct or return error if
 // the definition file has a bad section.
 func ParseDefinitionFile(r io.Reader) (d types.Definition, err error) {
-	d.Raw, err = io.ReadAll(r)
+	d.FullRaw, err = io.ReadAll(r)
+	defer func() {
+		d.StageRaw = d.FullRaw
```

Comment:
This needs an explanatory comment, that hints at where in the parsing flow this fires, and why it is needed.

Reply:
Oh, good catch - this is actually an artifact of a problem that was rendered moot by other changes. I'll replace it with some much more straightforward code.